ops/manifests/manipulations.py

```diff
@@ -138,10 +148,28 @@ def __call__(self, obj: AnyResource) -> None:
 class Addition(Manipulation):
     """Class used to define objects to add to the original manifests."""
 
-    def __call__(self) -> Optional[AnyResource]:
+    _called: Iterator[AnyResource] = iter(())
```

Comment:
Should we change this to an instance variable? I think this could (probably) generate odd side-effects if we use a class variable.

Reply:
aight, you got it